browser/ui/views/frame/brave_browser_view.h

```diff
@@ -128,6 +131,10 @@ class BraveBrowserView : public BrowserView,
   void OnWillBreakTile(const SplitViewBrowserData::Tile& tile) override;
   void OnSwapTabsInTile(const SplitViewBrowserData::Tile& tile) override;
 
+  auto secondary_contents_web_view() {
```

Comment:
This declaration seems to be at odds with the style guide [1] which indicates that `auto` deducted return types shouldn't be used on header files.

> use it only if the function or lambda has a very narrow scope, because functions with deduced return types don't define abstraction boundaries: the implementation is the interface. In particular, public functions in header files should almost never have deduced return types.

[https://google.github.io/styleguide/cppguide.html#Type_deduction]